src/lib/components/schedule/schedule-search-attributes.svelte

```diff
@@ -11,6 +11,13 @@
   $: indexedFields =
     decodedSearchAttributes?.searchAttributes.indexedFields ?? {};
   $: searchAttributeCount = Object.keys(indexedFields).length;
+
+  const formatValue = (value: Payload) => {
```

Comment:
I might pull this out into something like `payloadToString` or something, but I don't feel super strongly about this.

Reply:
[Done](https://github.com/temporalio/ui/pull/2420/commits/fa503cce3dc9adf4e3a895e116ce800e66eec361) ✔️